app/services/pairing_strategies/swiss.rb

```diff
@@ -69,13 +69,27 @@ def auto_score(pairing, player_index)
     end
 
     def apply_numbers!(sorter)
-      sorter.sort(round.pairings.non_bye).each_with_index do |pairing, i|
-        pairing.update(table_number: i + 1)
+      next_table = 1
+      sorter.sort(round.pairings.non_bye).each do |pairing|
```

Comment:
do you need to specify the sort field here to use automatic table?

Reply:
This is a method call to `PairingSorters::Ranked.sort`, it just takes the list of pairings. It's probably not necessary to pass the sorter in here. There was an alternative sort implementation before but I'll call it directly instead.

This isn't finished though.

Reply:
I've rewritten this now.

Reply:
sorry, i should have read more carefully.